specification/deviceprovisioningservices/data-plane/Microsoft.Devices/preview/2021-11-01-preview/examples/device/CertificateIssuance/getIssuedCertificate.json

```diff
@@ -0,0 +1,19 @@
+{
+  "parameters": {
+    "api-version": "2021-11-01-preview",
+    "idScope": "contoso-id-scope",
+    "registrationId": "contoso-registration-id",
+    "certificateId": "f679fd5a-a570-4d4b-b640-81753d47f820"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "issuedCertificate": "-----BEGIN CERTIFICATE-----MIIEsjCCApqgAwIBAgICEAEwDQYJKoZIhvcNAQELBQAwJjEkMCIGA1UEAwwbQ09OVE9TT19URVNUX0lOVEVSTUVESUFURV8xMB4XDTIyMDQyNjE3NTIzNVoXDTIyMDUyNjE3NTIzNVowLjEsMCoGA1UEAwwjY29udG9zby1yZWdpc3RyYXRpb24taWQuY29udG9zby5jb20wggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQDDUXRyR46cUts0KkBSrYHW9PNth2wcgzX5r5jxI2L1RHgFmcLLFiJEA52ilz/2ahfjo5+uzNa7xO66DUVc5ukPrfu/uXxBZc4sXMzJLK3+yNBQrvaSBixWFg10W6r5hmzBDbDkJRtq75DVjTfUhJRhYbG2PadKj/GKA4JLIijKBo8XJHjcEwesdLF2tWxrNt8QvcUEtT4Ge3SehoTH/AHSQLen2/qAxzKYNKt2sdJgTFZ5uJOUGDdOOF7GstVHcKsnHS1CgUW7EwsH2XLfZ67j35bpOOJOocD4Mg1CRS7FDRcH9pBZ+Cng0aH0mGflijWCB0HQ/zHk4zt9e3eg7UcTAgMBAAGjgeEwgd4wCQYDVR0TBAIwADARBglghkgBhvhCAQEEBAMCBkAwMwYJYIZIAYb4QgENBCYWJE9wZW5TU0wgR2VuZXJhdGVkIFNlcnZlciBDZXJ0aWZpY2F0ZTAdBgNVHQ4EFgQUALTpWEUQKgoM5nkVZgMxNoeB3HMwRQYDVR0jBD4wPIAUqV8aE5IM1sTp8YUMaBXxKxUDYK+hIKQeMBwxGjAYBgNVBAMMEUNPTlRPU09fVEVTVF9ST09UggIQADAOBgNVHQ8BAf8EBAMCBaAwEwYDVR0lBAwwCgYIKwYBBQUHAwEwDQYJKoZIhvcNAQELBQADggIBADOl1OSzOt7/I2GzY+NqM46ZcyDGWQsDlHq4I+fn6Jv0wX/x2wuMRjrh0WeT3gtUFD/MV0lqsk2O4YGvDavL3/XgwRbDBT9R0JaC86aCBBt6TUmq1g+lgx8Hh/ObllpZpYqbLIlFzWnuiFJonOv7fJ22y7Eta03wUZIJWqofXv5W1KKwscAxLrw0p6NWvwuRRKKtVp/wcJcBnjEh0uulHJYrgOiminPVmVfyWqht3VqZIesrHMrru5ITOVGpqLeSNpJnMBH3EQ6ClBCcPL6EDurIC/UgCUDMWjZKM2L4tNoaHwtFNo2KEBz7/4xloWix7+HzZdUL7Ly3WpFPk7UqOplKBKBni8KMbFMv8PgsKCu3/D8KTf1rHeEDhtRtOZrbi6yqxovD1YvLb8CqA6cNYn5+BiFKRGXCeXoI8l0qPGy4lXMRcUqdcYPD2gu98+vpa9zdB9OmLdhd7F3A2Y2yHULncaHKSFEnPmbE+sh+bg263gFXdoToVuvB5LeH1VPwaqINllCR6RyylKlKTF5Yy7N6NhvGbn5xxnFKvz7flgwdX1br4juJ430KeUCOzgkpyKoE0+WGQ91Vg5hkML27Ui90aLqQtlCClJwHalnOyQwvh8ZbGUdp0BJUbDhv0Bh5IF8Pkc7LLdMJUv/fEz3NDZyVJTpi23gdp/ebJhg5QbGE-----END CERTIFICATE-----",
```

Comment:
Is this actually loadable in most apps? In Key Vault, we encode line endings as well (either Windows or linux, but I recommend standardizing on a single set for ease) so that it's properly formatted PEM. If there's concerns about non-printable characters, base64- or base64url-encode.

Reply:
I will normalize these examples with a consistent line ending. Thanks for the feedback.